FocusEntity-Example/FocusEntity-Example/FocusARView.swift

```diff
@@ -17,8 +17,18 @@ class FocusARView: ARView {
   required init(frame frameRect: CGRect) {
     super.init(frame: frameRect)
     self.setupConfig()
-    self.focusEntity = FocusEntity(on: self, focus: .classic)
-//    self.focusEntity = FocusEntity(on: self, style: .colored(onColor: .red, offColor: .blue, nonTrackingColor: .orange))
+    //self.focusEntity = FocusEntity(on: self, focus: .classic)
+    //self.focusEntity = FocusEntity(on: self, focus: .plane)
+    do{
+        let onColor : MaterialColorParameter = try .texture(.load(named: "Add"))
+        let offColor : MaterialColorParameter = try .texture(.load(named: "Open"))
```

Comment:
bit nit-picky, but could you standardise the colons here removing that first space? (`onColor: Materi...`)